physionet-django/physionet/settings/base.py

```diff
@@ -534,6 +534,7 @@ class StorageTypes:
 
 SOURCE_CODE_REPOSITORY_LINK = config('SOURCE_CODE_REPOSITORY_LINK',
                                      default='https://github.com/MIT-LCP/physionet-build')
+MAX_TRAINING_REPORT_UPLOAD_SIZE = config('MAX_TRAINING_REPORT_UPLOAD_SIZE', cast=int, default=1)
```

Comment:
sorry, i should have spotted this before. default of 1 doesn't make sense if we are now using bytes.

Reply:
ah yes. my bad i missed this before. Updating asap

Reply:
@tompollard  do you think `default=1 * 1024 * 1024` would look better than `default= 1048576 ` 
Although ultimately both are the same , but the first one is much easier to change in future.

Reply:
hmm, i think i'd just go for `1048576`

Reply:
alright. sounds good